packages/utils/src/utils.ts

```diff
@@ -6,3 +6,17 @@ export function timer(fn: () => void): number {
   const end = performance.now() - start
   return end
 }
+
+/**
+ *
+ * @param input a string containing templated references to environment variables e.g. 'foo ${bar}'
+ * @returns a string with variables resolved to the actual values
+ */
+export function expand_env_vars(input: string) {
```

Comment:
should be camelCase

Reply:
why not pass in the variables, so we can reuse?

Something like

`expandTemplateString(input, process.env)`